taigaProject/src/taigaApi/task/getTaskHistory.py

```diff
@@ -53,6 +54,27 @@ def get_task_history(tasks, auth_token):
     return [cycle_time, closed_tasks]
 
 
+def get_lead_time(project_id, auth_token):
+    tasks = get_closed_tasks(project_id, auth_token)
+    lead_time = 0
+    closed_tasks = 0
+    lead_times = []
+    for task in tasks:
+
+        created_date = datetime.fromisoformat(task["created_date"])
+        finished_date = datetime.fromisoformat(task['finished_date'])
+        lead_time += (finished_date - created_date).days
+        lead_times.append({
+            "taskId": task["id"],
+            "startTime": task["created_date"],
+            "endTime": task['finished_date'],
+            "timeTaken": lead_time
```

Comment:
Should we rename this to leadTime? Time taken will not denote the unit, leaves ambiguity..